src/StaticWebAssetsSdk/Targets/Microsoft.NET.Sdk.StaticWebAssets.JSModules.targets

```diff
@@ -99,6 +99,11 @@ Copyright (c) .NET Foundation. All rights reserved.
          retrieve them.
     -->
 
+    <PropertyGroup>
+      <_ResolveJsInitializerModuleStaticWebAssetsCachePath>$(_StaticWebAssetsManifestBase)rjimswa.dswa.cache.json</_ResolveJsInitializerModuleStaticWebAssetsCachePath>
+    </PropertyGroup>
+
+
```

Comment:
Typo?

```suggestion
```

Reply:
I think this has been a bad merge on my part, thanks for catching it up